sig-multicluster/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Multicluster Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG Multicluster includes:
+
+- Solving common challenges related to management of multiple Kubernetes clusters and the applications that exist therein
+- Designing, discussing, implementing, and maintaining APIs, tools, and documentation related to multi-cluster administration and application management
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Mechanisms in Kubernetes that track information about multiple clusters
```

Comment:
Needs to be less vague.  spartakus tracks information about multiple clusters, in the sense that they all ping it... but I doubt that's under your purview